Scripts/Mac/BuildGVFSForMac.sh

```diff
@@ -50,3 +50,7 @@ $VFS_SCRIPTDIR/PublishGit.sh $GITPATH || exit 1
 
 echo 'Running VFS for Git unit tests...'
 $VFS_PUBLISHDIR/GVFS.UnitTests || exit 1
+
+echo 'Creating installer package...'
```

Comment:
Ideally, this step should be wrapped in a csproj file that is then included in GVFS.sln, similar to what we did on Windows (I would rename the existing project to `GVFS.Installer.Windows` and create a new one called `GVFS.Installer.Mac`). Our "source of truth" for the build should be the sln file, not these build scripts, though that's not 100% satisfied at the moment. This is something @jrbriggs  is working on cleaning up.